cmd/vic-machine/configure/configure.go

```diff
@@ -190,6 +191,12 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 		updateSessionEnv(vicAdminSession, config.VICAdminNoProxy, nProxy)
 	}
 
+	if c.ComputeResourcePath != "" {
+		updateSessionEnv(portlayerSession, config.PortLayerCSPath, v.Session().Cluster.InventoryPath)
```

Comment:
How about if I only specified the name instead of full inventory path for compute resource ?

Reply:
ComputeResourcePath is read from command line, so it can either a cluster name or a inventory path name, and we will get its inventory path in either case and save into v.Session().